utils/db-parts.stanza

```diff
@@ -4,11 +4,16 @@ defpackage ocdb/db-parts :
   import collections
   import json
   import math
+  import lang-utils
```

Comment:
We should be careful with this import since it is very useful but the interfaces are undocumented.